content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -153,109 +153,153 @@ Once you have a Linux-based Kubernetes master node you are ready to choose a net
+    ```
+    where `-ConfigFile` points to the path of the Kubernetes configuration file.
 
-```PowerShell
-cd c:\k
-[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-wget https://raw.githubusercontent.com/Microsoft/SDN/master/Kubernetes/flannel/start.ps1 -o c:\k\start.ps1
-```
+![alt_text](../kubecluster.ps1-join.gif "KubeCluster.ps1 join output")
 
 {{< note >}}
-[start.ps1](https://github.com/Microsoft/SDN/blob/master/Kubernetes/flannel/start.ps1) references [install.ps1](https://github.com/Microsoft/SDN/blob/master/Kubernetes/windows/install.ps1), which downloads additional files such as the `flanneld` executable and the [Dockerfile for infrastructure pod](https://github.com/Microsoft/SDN/blob/master/Kubernetes/windows/Dockerfile) and install those for you. For overlay networking mode, the [firewall](https://github.com/Microsoft/SDN/blob/master/Kubernetes/windows/helper.psm1#L111) is opened for local UDP port 4789. There may be multiple powershell windows being opened/closed as well as a few seconds of network outage while the new external vSwitch for the pod network is being created the first time. Run the script using the arguments as specified below:
+ Should the script fail during the bootstrap or joining procedure for whatever reason, consecutive join attempts should be done in a fresh Powershell window.  
 {{< /note >}}
 
-```PowerShell
-.\start.ps1 -ManagementIP <Windows Node IP> -NetworkMode overlay  -ClusterCIDR <Cluster CIDR> -ServiceCIDR <Service CIDR> -KubeDnsServiceIP <Kube-dns Service IP> -LogDir <Log directory>
-```
-
-| Parameter | Default Value | Notes |
-| --- | --- | --- |
-| -ManagementIP | N/A (required) | The IP address assigned to the Windows node. You can use `ipconfig` to find this. |
-| -NetworkMode | l2bridge | We're using `overlay` here |
-| -ClusterCIDR | 10.244.0.0/16 | Refer to your cluster IP plan |
-| -ServiceCIDR | 10.96.0.0/12 | Refer to your cluster IP plan |
-| -KubeDnsServiceIP | 10.96.0.10 | |
-| -InterfaceName | Ethernet | The name of the network interface of the Windows host. You can use <code>ipconfig</code> to find this. |
-| -LogDir | C:\k | The directory where kubelet and kube-proxy logs are redirected into their respective output files. |
+This step will perform the following actions:
+  * Establish SSH session with the Kubernetes control-plane ("Master") node to retrieve the [Kubeconfig file](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) for cluster access 
+  * Register Kubelet as a Windows service
+  * Configure CNI network plugins
+  * Create a HNS network on top of the chosen network interface
+      {{< note >}}
+      This may cause a network blip for a few seconds while the vSwitch is being created.
+      {{< /note >}}
+  * (If vxlan plugin is selected) Open up inbound firewall UDP port 4789 for overlay traffic
+  * Register FlannelD as a Windows service
+  * Register Kube-proxy as a Windows service
```

Comment:
some of the component names in the above list probably should be lowercased as in the rest of the k8s.io documentation - e.g. kube-proxy, kubelet.

Reply:
addressed in the newest commit, thank you!